mysite/profile/tasks/__init__.py

```diff
@@ -88,37 +88,19 @@ class FetchPersonDataFromOhloh:
     name = "profile.FetchPersonDataFromOhloh"
 
     def run(self, dia_id, **kwargs):
```

Comment:
@onceuponatimeforever Should the dia_id be removed from the function definition?